code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -177,563 +216,742 @@
 
 	bobcurl
 		name = "Bobcurl"
 		icon_state = "hair_bobcurl"
 		gender = FEMALE
 		species_allowed = list("Human","Unathi")
+		chatname = "Bobbed Hair"
 
 	bob
 		name = "Bob"
 		icon_state = "hair_bobcut"
 		gender = FEMALE
 		species_allowed = list("Human","Unathi")
+		chatname = "Bobbed Hair"
 
 	bobcutalt
 		name = "Chin Length Bob"
 		icon_state = "hair_bobcutalt"
 		gender = FEMALE
+		chatname = "Chin Length Bob Haircut"
 
 	bowl
 		name = "Bowl"
 		icon_state = "hair_bowlcut"
 		gender = MALE
+		chatname = "Bowl Cut"
 
 	buzz
 		name = "Buzzcut"
 		icon_state = "hair_buzzcut"
 		gender = MALE
 		species_allowed = list("Human","Unathi")
+		chatname = "Short Hair"
 
 	crew
 		name = "Crewcut"
 		icon_state = "hair_crewcut"
 		gender = MALE
+		chatname = "Short Hair"
 
 	combover
 		name = "Combover"
 		icon_state = "hair_combover"
 		gender = MALE
+		chatname = "Short Hair"
 
 	reversemohawk
 		name = "Reverse Mohawk"
 		icon_state = "hair_reversemohawk"
 		gender = MALE
+		chatname = "Mohawk"
 
 	devillock
 		name = "Devil Lock"
 		icon_state = "hair_devilock"
+		chatname = "Devil Locks"
 
 	dreadlocks
 		name = "Dreadlocks"
 		icon_state = "hair_dreads"
+		length = 4
+		chatname = "Dreadlocks"
 
 	curls
 		name = "Curls"
 		icon_state = "hair_curls"
+		name = "Curled Hair"
 
 	afro
 		name = "Afro"
 		icon_state = "hair_afro"
+		length = 4
+		chatname = "Afro"
 
 	afro2
 		name = "Afro 2"
 		icon_state = "hair_afro2"
+		length = 4
+		chatname = "Afro"
 
 	afro_large
 		name = "Big Afro"
 		icon_state = "hair_bigafro"
 		gender = MALE
+		length = 4
+		chatname = "Big Afro"
 
 	rows
 		name = "Rows"
 		icon_state = "hair_rows1"
+		length = 2
+		chatname = "Cornrows"
 
 	rows2
 		name = "Rows 2"
 		icon_state = "hair_rows2"
+		length = 2
+		chatname = "Cornrows"
 
 	sargeant
 		name = "Flat Top"
 		icon_state = "hair_sargeant"
 		gender = MALE
+		chatname = "Short Hair"
 
 	emo
 		name = "Emo"
 		icon_state = "hair_emo"
+		chatname = "Short Hair"
 
 	longemo
 		name = "Long Emo"
 		icon_state = "hair_emolong"
 		gender = FEMALE
+		length = 2
+		chatname = "Short Hair"
 
 	fringeemo
 		name = "Emo Fringe"
 		icon_state = "hair_emofringe"
+		chatname = "Fringed Hair"
 
 	shortovereye
 		name = "Overeye Short"
 		icon_state = "hair_shortovereye"
+		chatname = "Short Hair"
 
 	veryshortovereyealternate
 		name = "Overeye Very Short, Alternate"
 		icon_state = "hair_veryshortovereye"
+		chatname = "Short Hair"
 
 	longovereye
 		name = "Overeye Long"
 		icon_state = "hair_longovereye"
+		length = 2
+		chatname = "Long Hair"
 
 	fag
 		name = "Flow Hair"
 		icon_state = "hair_f"
+		length = 2
+		chatname = "Flowing Hair"
 
 	feather
 		name = "Feather"
 		icon_state = "hair_feather"
+		length = 2
+		chatname = "Feathered Hair"
 
 	hitop
 		name = "Hitop"
 		icon_state = "hair_hitop"
 		gender = MALE
+		chatname = "Hitop"
 
 	jensen
-		name = "Adam Jensen Hair"
+		name = "Jensen Hair"  // Removing Videogame References

```

Comment:
If you're going to do this, you do realize that this is still a reference? Not that I really believe hairstyles need to be dereferenced, because even if you rename them the hairstyle itself is still a reference...

Reply:
why even remove the reference, its a fun gag?